### PR TITLE
[Outlook] (append on send) Add conceptual article

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[730ab81](https://github.com/OfficeDev/office-js-docs-pr/commits/730ab8182d07937929cda6a76ac9b7f62b1b9538)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/append-on-send.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/outlook/append-on-send.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C21%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005212236533449-1881%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Y4P9JzvH56MuLgygBy6GZWjVo72b5k1DXyS7nYkTv0g%3D&st=2020-05-21T22%3A33%3A36Z&se=2020-06-21T22%3A38%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a3a2000](https://github.com/OfficeDev/office-js-docs-pr/commits/a3a2000107fd163368801148edd8dc05a9ad2e2f)_: 

:clock10: **Pending**: waiting for processors ([55](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 55 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a3a2000](https://github.com/OfficeDev/office-js-docs-pr/commits/a3a2000107fd163368801148edd8dc05a9ad2e2f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/append-on-send.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/outlook/append-on-send.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005262017160956-1881%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=mTEZtsQ3uSexQLsX0AOpfCKa31gv24vXmOLRbEfaMts%3D&st=2020-05-26T20%3A15%3A32Z&se=2020-06-26T20%3A20%3A32Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ec5e82b](https://github.com/OfficeDev/office-js-docs-pr/commits/ec5e82b9beb753d0923e4c244fd8690def5ae8ff)_: 

:clock10: **Pending**: waiting for processors ([42](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 42 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ec5e82b](https://github.com/OfficeDev/office-js-docs-pr/commits/ec5e82b9beb753d0923e4c244fd8690def5ae8ff)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/append-on-send.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/outlook/append-on-send.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005262119542114-1881%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=QY8OsY0mhprN8RiLJNXxL4GVw8FvtxA86ULhmqU0LqE%3D&st=2020-05-26T21%3A18%3A15Z&se=2020-06-26T21%3A23%3A15Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[015320a](https://github.com/OfficeDev/office-js-docs-pr/commits/015320ab11e842180ab688d6d1820c0bfde16665)_: 

:clock10: **Pending**: waiting for processors ([19](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 19 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by o365devx** [comment]
Docs Build status updates of commit _[015320a](https://github.com/OfficeDev/office-js-docs-pr/commits/015320ab11e842180ab688d6d1820c0bfde16665)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/append-on-send.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/outlook/append-on-send.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005262142361270-1881%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=38X38QKOirgGDd%2BNJ%2BJuiW3jDGXdm739ZNILPHKAuMU%3D&st=2020-05-26T21%3A39%3A40Z&se=2020-06-26T21%3A44%3A40Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[2236ccc](https://github.com/OfficeDev/office-js-docs-pr/commits/2236ccc6a4a1441af16e4ec222c68876910d15ff)_: 

:clock10: **Pending**: waiting for processors ([12](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 12 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by o365devx** [comment]
Docs Build status updates of commit _[2236ccc](https://github.com/OfficeDev/office-js-docs-pr/commits/2236ccc6a4a1441af16e4ec222c68876910d15ff)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/outlook-web-append-disclaimer.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/images/outlook-web-append-disclaimer.png) | :white_check_mark:Succeeded | |
[docs/outlook/append-on-send.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/outlook/append-on-send.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005262220251133-1881%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ad%2BnGW0fg7rBuQPnSrl0Dm%2BPIbvWrzhNve5oEWLwotw%3D&st=2020-05-26T22%3A18%3A58Z&se=2020-06-26T22%3A23%3A58Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/outlook/append-on-send.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+---
+title: Implement append on send in your Outlook add-in (preview)
+description: Learn how to implement the append-on-send feature in your Outlook add-in.
```
Comment (lindalu-MSFT) [inline docs/outlook/append-on-send.md:3]:
```suggestion
description: Learn how to implement the append on send feature in your Outlook add-in.
```
Hi, see my comments in Teams but I think we should be consistent here. We've already published "append on send" without hyphens. Suggest you remove throughout the doc, or fix all in the docset. Also, consider referring to as AppendOnSend?

File: docs/outlook/append-on-send.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+ms.topic: article
+ms.date: 05/26/2020
+localization_priority: Normal
+---
+
+# Implement append on send in your Outlook add-in (preview)
+
+By the end of this walkthrough, you'll have an add-in that can insert a disclaimer when a message is sent.
```
Comment (lindalu-MSFT) [inline docs/outlook/append-on-send.md:11]:
```suggestion
By the end of this walkthrough, you'll have an Outlook add-in that can insert a disclaimer when a message is sent.
```
FYI, Google crawls the first sentence of the first paragraph so always be sure to add SEO terms ("tags").

File: docs/outlook/append-on-send.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+> [!IMPORTANT]
+> This feature is only supported for [preview](../reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) in Outlook on the web and Windows with an Office 365 subscription. See [How to preview the append-on-send feature](#how-to-preview-the-append-on-send-feature) in this article for more details.
```
Comment (lindalu-MSFT) [inline docs/outlook/append-on-send.md:14]:
```suggestion
> This feature is currently supported for [preview](../reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) in Outlook on the web and Windows with an Office 365 subscription. See [How to preview the append-on-send feature](#how-to-preview-the-append-on-send-feature) in this article for more details.
```

File: docs/outlook/append-on-send.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+1. At the end of the file, add the following statement.
+
+    ```js
+    g.appendDisclaimerOnSend = appendDisclaimerOnSend;
+    ```
+
+## Try it out
+
+1. Run the following command in the root directory of your project. When you run this command, the local web server will start (if it's not already running).
```
Comment (lindalu-MSFT) [inline docs/outlook/append-on-send.md:174]:
```suggestion
1. Run the following command in the root directory of your project. When you run this command, the local web server will start, if it's not already running.
```

File: docs/outlook/append-on-send.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+
+    ```command&nbsp;line
+    npm run dev-server
+    ```
+
+1. Follow the instructions in [Sideload Outlook add-ins for testing](sideload-outlook-add-ins-for-testing.md) to sideload the add-in in Outlook.
```
Comment (lindalu-MSFT) [inline docs/outlook/append-on-send.md:180]:
```suggestion
1. Follow the instructions in [Sideload Outlook add-ins for testing](sideload-outlook-add-ins-for-testing.md).
```
redundant

File: docs/outlook/append-on-send.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+
+1. In Outlook, create a new message and add yourself to the **To** line.
```
Comment (lindalu-MSFT) [inline docs/outlook/append-on-send.md:182]:
```suggestion
1. Create a new message, and add yourself to the **To** line.
```
While technically you do not need the comma before the conjunction, for procedures we add them more aggressively between clauses to assist machine localization.

File: docs/outlook/append-on-send.md [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+
+1. Choose the **Perform an action** button from the ribbon or overflow menu.
```
Comment (lindalu-MSFT) [inline docs/outlook/append-on-send.md:184]:
```suggestion
1. From the ribbon or overflow menu, choose **Perform an action**.
```

File: docs/outlook/append-on-send.md [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+
+1. Send the message then open it from your **Inbox** or **Sent Items** folder to view the appended disclaimer.
```
Comment (lindalu-MSFT) [inline docs/outlook/append-on-send.md:186]:
```suggestion
1. Send the message, then open it from your **Inbox** or **Sent Items** folder to view the appended disclaimer.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[494a387](https://github.com/OfficeDev/office-js-docs-pr/commits/494a387dfe2b98d6f32abf83afef84685f605448)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/outlook-web-append-disclaimer.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/images/outlook-web-append-disclaimer.png) | :white_check_mark:Succeeded | |
[docs/outlook/append-on-send.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/outlook/append-on-send.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-append-on-send/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C5%5C27%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202005271814501817-1881%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=mkUojzi2%2Bw2r4eYMHCpzGTAdfRJ05oAq6jIaKw%2FtHwE%3D&st=2020-05-27T18%3A11%3A30Z&se=2020-06-27T18%3A16%3A30Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**